ksqldb-rest-app/src/test/java/io/confluent/ksql/api/CorsTest.java

```diff
@@ -251,7 +251,7 @@ private void shouldAcceptPreflightRequest(final String origin, final String allo
         MultiMap.caseInsensitiveMultiMap().add("origin", origin)
             .add(ACCESS_CONTROL_REQUEST_METHOD_HEADER, "POST"));
 
-    assertThat(response.statusCode(), is(200));
+    assertThat(response.statusCode(), is(204));
```

Comment:
Though the change is matching. Not sure it is standard practice for `OPTIONS` call. [fyi]